src/login/components/GroupManagement/Groups/EditGroup.js

```diff
@@ -6,11 +6,11 @@ const EditGroup = (props) => {
   return (
     <div className="edit-data">
       <div className="title">
-        <p>Edit your group {props.group.group.display_name}</p>
+        <p>Edit your group {props.group.display_name}</p> 
```

Comment:
this one need to be reverted to the previous   "props.group.group.display_name". the updated "props.group.display_name" is undefined.